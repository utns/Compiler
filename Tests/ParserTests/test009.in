procedure foo();
begin
end;

begin
    foo();
end.