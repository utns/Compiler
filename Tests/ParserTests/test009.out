begin
end

begin
(
__foo
end
